bundle/manifests/nexus-operator.clusterserviceversion.yaml

```diff
@@ -32,69 +32,37 @@ metadata:
           }
```

Comment:
Should this file have been changed at all? It's 0.2

Reply:
This file does not exist any more, GH is accusing a rename, but was deleted actually.